modules/installation-creating-gcp-worker.adoc

```diff
@@ -39,11 +43,31 @@ have to contact Red Hat support with your installation logs.
 section of this topic and save it as `06_worker.py` on your computer. This
 template describes the worker machines that your cluster requires.
 
-. Export the following variables needed by the resource definition:
+. Export the variables that the resource definition uses.
+.. Export the subnet that hosts the compute machines:
 +
+ifndef::shared-vpc[]
 ----
 $ export COMPUTE_SUBNET=`gcloud compute networks subnets describe ${INFRA_ID}-worker-subnet --region=${REGION} --format json | jq -r .selfLink`
+----
+endif::shared-vpc[]
+ifdef::shared-vpc[]
+----
+$ export COMPUTE_SUBNET=`gcloud compute networks subnets describe <worker-subnet> --region=${REGION} --format json | jq -r .selfLink`
+----
++
+Where `<worker-subnet>` is the name of the subnet in the shared VPC to deploy the compute machines to.
+endif::shared-vpc[]
+
+.. Export the email address for your service account:
++
+----
 $ export WORKER_SERVICE_ACCOUNT_EMAIL=`gcloud iam service-accounts list | grep "^${INFRA_ID}-worker-node " | awk '{print $2}'`
```

Comment:
It seems duplicate with WORKER_SA

Reply:
This was an existing command and might be a bug. Do you know which command should be used to generate the worker service email address?

Reply:
Yeah, I've logged a bz Bug 1852450 - GCP UPI WORKER_SERVICE_ACCOUNT_EMAIL is duplicate with WORKER_SA and MASTER_SERVICE_ACCOUNT_EMAIL is duplicate with MASTER_SA. It's okay to not fix at this point.